src/appsco-billing-page.html

```diff
@@ -782,6 +784,13 @@ <h2 class="invoice-list-title">Invoice list</h2>
                 }
             },
 
+            _onCompanyApiChanged: function() {
+                if(this.companyApi) {
```

Comment:
What if 'this.companyApi' exists but is empty string? It will still call bad API link.